editor.planx.uk/src/pages/FlowEditor/lib/analyticsProvider.tsx

```diff
@@ -296,6 +307,39 @@ export const AnalyticsProvider: React.FC<{ children: React.ReactNode }> = ({
     }
   }
 
+  async function trackBackwardsNavigationByNodeId(
+    nodeId: string,
+    initiator: BackwardsNaviagtionInitiatorType,
+  ) {
+    const targetNodeMetadata = getTitleAndTypeFromFlow(nodeId);
+    console.log("Target node info: ", targetNodeMetadata);
+    const metadata: Record<string, NodeMetadata> = {};
+    metadata[`${initiator}`] = targetNodeMetadata;
```

Comment:
I opted for this syntax as I was getting errors that the `initiator` wasn't being used when I was trying to apply is a key in the object creation.